net.puddletag.puddletag.appdata.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>net.puddletag.puddletag</id>
+  <launchable type="desktop-id">net.puddletag.puddletag.desktop</launchable>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  
+  <name>Puddletag</name>
+  <summary>A powerful ID3 tag editor</summary>
+
+  <releases>
+    <release date="2022-02-05" version="2.1.1" />
+  </releases>
+
+  <categories>
+    <category>Audio & Video</category>
```

Comment:
a bare amperand in XML is always bad. This is what causes the build failure.

Also I'm not even sure this is a valid category.
Valid categories are https://specifications.freedesktop.org/menu-spec/latest/apa.html

Reply:
Yup, fixed.